lib/bolt/config/options.rb

```diff
@@ -475,6 +477,12 @@ module Options
           _plugin: true,
           _example: { "cleanup" => false, "tmpdir" => "/tmp/bolt" }
         },
+        "lxd" => {
+          description: "A map of configuraiton options for the LXD transport",
+          type: Hash,
+          _plugin: true,
+          _example: {}
```

Comment:
```suggestion
          _example: { "cleanup": false }
```

This should have an example so the example file for `bolt-defaults.yaml` isn't just an empty hash.